cypress/integration/Tooltip.spec.ts

```diff
@@ -155,14 +155,14 @@ describe('Tooltip', () => {
     });
 
     it('the span element should not have an aria-describedby attribute', () => {
-      cy.get('button').should('not.have.attr', 'aria-describedby');
+      cy.get('span').should('not.have.attr', 'aria-describedby');
```

Comment:
This was caught by the change in Cypress v6. Before, `not.have.attr` would pass even if `cy.get('button')` never actually matched anything. This was a typo from long ago that Cypress was perfectly happy to incorrectly pass.